code-reviews/recipes/Markdown.md

```diff
@@ -0,0 +1,175 @@
+# Markdown Code Reviews
+
+## Style Guide
+
+[CSE](../../CSE.md) developers treat documentation like other source code and follow the same rules and checklists when reviewing documentation as code.
+
+Documentation should both use good markdown syntax to ensure it's properly parsed, and follow good [writing style guidelines](#writing-style) to ensure the document is easy to read and understand.
+
+## Linting
+
+Linting tools exist both for verifying proper markdown syntax as well as grammar and proper English language.
+
+A good setup includes a markdown linter used during editing and PR build verification, and a grammar linter used while editing the document.
```

Comment:
Should we mention that example linters are referenced below?

Reply:
Fixed